Xamarin.Forms.Platform.Android/Renderers/TimePickerRenderer.cs

```diff
@@ -164,8 +165,17 @@ void OnCancelButtonClicked(object sender, EventArgs e)
 
 		void SetTime(TimeSpan time)
 		{
-			var timeFormat = Is24HourView ? "HH:mm" : Element.Format;
-			EditText.Text = DateTime.Today.Add(time).ToString(timeFormat);
+			if (Element.Format.Equals(""))
```

Comment:
Can this use? `String.IsNullOrEmpty` 

If not you can also do
`if (Element.Format.Equals(String.Empty))`